swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -246,8 +247,88 @@ Follow the steps below to obtain API tokens for Google Sheets.
+    3. For step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use. 
+    4. Proceed with the necessary Google account validation steps. 
+    5. Click **Exchange authorization code for tokens** to obtain the refresh and access tokens.
+
+## Redis [Experimental]
+
+The Redis data store is a key-value-based inmemory datastore useful for storing and retrieving unstructured data quickly. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
+
+```shell
+$ bal persist init
+```
+
+### Supported Ballerina types
+
+The Redis data store supports the following Ballerina types. As we have utilized `hash` and `set` as Redis data types to store objects, the following data types will be converted to `string`.
+
+- int
+- float
+- decimal
+- string
+- time:Date
+- time:TimeOfDay
+- time:Utc
+- time:Civil
+- enum
+
+### Configuration
+
+You need to set values for the following basic configuration parameters in the `Config.toml` file in your project to use the Redis data store.
+
+| Parameter  |             Description              |
+|:----------:|:------------------------------------:|
+| connection | The connection URI of the DB server. |
+
+The following is a sample `Config.toml` file with the Redis data store configuration. This is generated by the `persist generate` command.
+
+```toml
+[<packageName>.<moduleName>.redis]
+connection = "redis://localhost:6379"
+```
+
+Alternatively, you can provide connection parameters instead of the `URI` as follows.
+
+```toml
+[<packageName>.<moduleName>.redis.connection]
+host = "localhost"
+port = 6379
+```
+
+Additionally, you can set values for the advanced configuration parameters in the `Config.toml` file in your project to use the Redis data store. For more information on these parameters, see the [Redis Connector documentation](https://central.ballerina.io/ballerinax/redis/latest#ConnectionConfig).
+
+### Setup guide
+
+#### Setup a Redis server instance
+
+Select one of the methods below to set up a Redis server.
+
+- Install a Redis server on your machine locally by downloading and installing it based on your development platform. See the [official Redis documentation](https://redis.io/download/).
+- Use Docker to create a DB server deployment.
+   1. Install Docker on your machine if you haven't already.
+   2. Pull the Redis Docker image from Docker Hub using the following command.
+
+      ```
+      $ docker pull redis
+      ```
+   
+   3. Run the Redis container as follows.
+   
+      ```
+      $ docker run -d -p 6379:6379 --name <container-name> redis
+      ```
+   
+- Use a cloud-based DB solution such as Google’s Cloud, Amazon’s Web Services, or Microsoft’s Azure database.
+   1. Visit [Redis cloud console](https://app.redislabs.com).
+   2. Log in using email and password or using one of the single sign-On options.
```

Comment:
```suggestion
   2. Log in using email and password or using one of the single sign-on options.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/8838/commits/4612960007eaacc7c43ef86420f1bddaf4b13c74